### ISSUE TITLE
[Quest] Zorbin's Ultra-Shrinker is usable an non-giant NPCs

### ISSUE THREAD

**Issue by ishidinho** [chromiecraft/chromiecraft#2542]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

### Current Behaviour

During the quests "Zapped Giants" and "Again With the Zapped Giants" in Feralas, the item [Zorbin's Ultra-Shrinker] is usable on non-giant NPCs that are not related to the quest.

![WoWScrnShot_121021_185553](https://user-images.githubusercontent.com/62797536/145619772-fa4d47f7-9f13-400a-b68e-e60b38e2591b.jpg)

### Expected Blizzlike Behaviour

The item [Zorbin's Ultra-Shrinker] should only be usable in NPCs related to the quest "Zapped Giants" and "Again With the Zapped Giants" that can drop the quest item [Miniaturization Residue].

### Steps to reproduce the problem

1. Accept quest "Zapped Giants" in Feralas
2. Use quest item [Zorbin's Ultra-Shrinker] on NPCs that are not related to the quest

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/0300cef119ea1708f26ab1519d46c652410b414f

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Shineslippers** [comment]
Yes you can use it on other creatures, but it doesn't affect them. that's not ehought for report.
